docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/kie-server-was-environment-set-proc.adoc

```diff
@@ -52,17 +52,33 @@ ifdef::DM[]
 endif::DM[]
 
 ifdef::BA[]
+|`kie.services.jms.queues.response`
+|`jms/KIE.RESPONSE.ALL`
+|The JNDI name of JMS queue for responses used by the {CENTRAL} and {KIE_SERVER}.
+
+|`org.uberfire.domain`
```

Comment:
it should be org.kie.server.domain instead of org.uberfire.domain as that one was used in business central